### PR TITLE
TELCODOCS-386: SRO metrics RNs

### PR DESCRIPTION
**Pull request by cherraz**
RNs for : https://issues.redhat.com/browse/TELCODOCS-386

Preview: https://deploy-preview-41567--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-scalability-and-performance

Versions: 4.10 only

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -292,8 +292,13 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-monitoring"]
 === Monitoring
 
-[id="ocp-4-10-scale"]
-=== Scale
+[id="ocp-4-10-scalability-and-performance"]
+=== Scalability and performance
+
+==== New Special Resource Operator metrics
```
Comment (ctauchen) [inline release_notes/ocp-4-10-release-notes.adoc:298]: Your L4 heading needs a unique id.

Something like this:

`[id="ocp-4-10-SRO-metrics]`

**Review by ctauchen — COMMENTED** [review]
Just one comment about a missing id. Otherwise LGTM!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (cherraz) [reply release_notes/ocp-4-10-release-notes.adoc:298]: OH I didn't realize that was a thing. Will add, thank you :)

**Review by liqcui — COMMENTED** [review]
/lgtm

**Pull request merged**